tests/Subutai.Repository.SqlRepository.Tests/ProjectEntityRepositoryTests.cs

```diff
@@ -89,4 +90,93 @@ public async Task AddAsync_ShouldHandleException()
         // Assert
         await act.Should().ThrowAsync<Exception>();
     }
+    #endregion
+
+    #region update method test
+    [Fact]
+    public async Task UpdateAsync_ShouldUpdateProjectEntity()
+
+    {
+        // Arrange
+        var existingEntity = new ProjectEntity
+        {
+            Id = 1,
+            Name = "Old Name"
+        };
+
+        _context.Projects.Add(existingEntity);
+        await _context.SaveChangesAsync();
+
+        var updateEntity = new ProjectEntity
+        {
+            Id = existingEntity.Id,
+            Name = "New Name"
+        };
+
+        // Act
+        var result = await _repository.UpdateAsync(updateEntity);
+
+        // Assert
+        result.Should().NotBeNull();
+        result.Id.Should().Be(existingEntity.Id);
+        result.Name.Should().Be(updateEntity.Name);
```

Comment:
[Suggestion]

Multiple assertions should be moved into AssertionScope

```suggestion
using (new AssertionScope())
{
        result.Should().NotBeNull();
        result.Id.Should().Be(existingEntity.Id);
        result.Name.Should().Be(updateEntity.Name);
}
```

Reply:
I have checked and found it in 2 places, I put it in the using scope and fixed it. thank you :)